fedora/product.yml

```diff
@@ -11,7 +11,12 @@ pkg_manager: "dnf"
 init_system: "systemd"
 
 # The fingerprints below are retrieved from https://getfedora.org/keys/
-latest_version: 28
-latest_release_fingerprint: "128C F232 A937 1991 C8A6  5695 E08E 7E62 9DB6 2FB1"
-previous_version: 27
-previous_release_fingerprint: "860E 19B0 AFA8 00A1 7518  81A6 F55E 7430 F528 2EE4"
+latest_version: 29
+latest_release_fingerprint: "5A03B4DD8254ECA02FDA1637A20AA56B429476B4"
+latest_pkg_release: 5a886537
```

Comment:
I would add quotes here, so the version string is not confused with a number by the YAML parser.

Reply:
Nice catch, made them strings.